src/main.js

```diff
@@ -28,4 +28,8 @@ exports.toJS = helpers.toJS
 exports.toImmutable = helpers.toImmutable
 exports.isImmutable = helpers.isImmutable
 
-exports.createReactMixin = require('./create-react-mixin')
+// @todo: improve how to load these addons
+// because react is defined as externals in webpack, not huge loss of space
+// if they aren't used
+exports.ReactMixin = require('../addons/react-mixin')
```

Comment:
I've been thinking the way we want to require these is via `require('nuclear-js/addons/react-mixin')` as this will make it totally opt-in for people that dont use React.